### DISCUSSION TITLE
plugin, pc, g6, element, core 之间有什么关系？如何进行相辅相成的？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [antvis/G6#4233] [Q&A] [answered]
plugin, pc, g6, element, core 之间有什么关系？它们之间如何进行相辅相成的？

**Comment by hustcc** [comment, marked as answer]
新的 G6 5.0 架构中，已经去掉了这些不合理的设计，全部分成为 G6 + extensions 两类，其中 extensions 会有不同类型的插件的不同包。
